Status/Day_11.md

```diff
@@ -54,6 +54,21 @@ Solution by: CoffeeBrakeInc
 tup = (1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
 lt = int(len(tup)/2)
 print(tup[:lt], tup[lt:])
+```
+------------------
+**OR**
+```python
+
+'''
+Solution by: AasaiAlangaram
+'''
+
+tp = (1,2,3,4,5,6,7,8,9,10)
+
+print('The Original Tuple:',tp)
+
+splitted = [print('Splitted List :{List}'.format(List = tp[x:x+5])) for x in range(0,len(tp),5)]
```

Comment:
The solution is quite very unique and nice. Here, the program will satisfy the output even if the variable **splitted** is not in there. To reduce redundancy, I'm modifying this line. Hope, it will be okay for you :)